flink-end-to-end-tests/flink-end-to-end-tests-common-kafka/src/test/java/org/apache/flink/tests/util/kafka/SQLClientKafkaITCase.java

```diff
@@ -76,7 +77,9 @@
 	}
 
 	@Rule
-	public final FlinkResource flink = new LocalStandaloneFlinkResource();
+	public final FlinkResource flink = new LocalStandaloneFlinkResourceFactory()
+		.create(FlinkResourceSetup.builder().build())
+		.get();
```

Comment:
These tests have are in some form or another reliant on the cluster running locally. In the case of this test IIRC it was because the input CSV file exists locally; for the metrics IT case it is because it assumes the JM to be on the local host.

Reply:
nit: shorthand for `LocalStandaloneFlinkResourceFactory`: `FlinkResource#getLocal`